aten/src/ATen/native/layer_norm.h

```diff
@@ -106,6 +106,12 @@ void layer_norm_cpu_out(
     int64_t M,
     int64_t N);
 
+std::tuple<Tensor, Tensor> rms_norm_cpu(
```

Comment:
Hmm, why this one is needed? Should there be rms_norm just dispatch to native, otherwise you going to generic?